addons/l10n_fr_invoice_addr/views/account_move_views.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="account_move_form_l10n_fr_invoice_addr" model="ir.ui.view">
+        <field name="name">l10n_fr_invoice_addr.account.move.form</field>
+        <field name="model">account.move</field>
+        <field name="inherit_id" ref="account.view_move_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='partner_shipping_id']" position="attributes">
+                <attribute name="groups"/>
+                <attribute name="attrs">
+                    {'invisible': [
+                        ('move_type', 'not in', ('out_invoice', 'out_refund', 'out_receipt')),
```

Comment:
This is the original condition that makes things (in)visible. Since I just want to add a case where it's visible (when the company is french), I've copied over this original condition